### ISSUE TITLE
audit-avt1-overview: repeated alt value in same link

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon-website#1545]
## Environment:

> Operating system: Mac
> Browser: Chrome 
> AVT :one:
> Page: [Components - Overview](https://www.carbondesignsystem.com/components/overview/)
> This issue accounts for 34 total violations

## Detailed Description: 

Link text is repeated in an image 'alt' value within the same link

<img width="1680" alt="Screen Shot 2020-08-11 at 12 06 45 PM" src="https://user-images.githubusercontent.com/32720851/89922632-4f186980-dbcd-11ea-9b47-585c5fe19a41.png">

## What to do: 
If a link has text and a redundant image, set the image alt attribute to alt=“” (null alt without any spaces);

## Why is this important?
When the text equivalent of an image inside or adjacent to a link is the same as the link text, people with visual impairments who use a screen reader will hear the same text announced twice. Also those who use text only browsers will see both the link text and text equivalent of the image, and can be confused by the redundant text.
